src/components/molecules/Cv/components/BasicData.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { Box } from '../../../quarks';
+import { CvHeading, CvDescription } from '../../../atoms';
+import { ICvTypes } from '../Cv.types';
+
+export const BasicData: FC<IBasicDataProps> = ({
+  studentStatus = '',
+  workingHours = '',
+  sanitationBook = '',
+  drivingLicences = [],
+}) => {
+  return (
+    <Box>
+      <CvHeading heading="Informacje podstawowe:" />
+      <CvDescription basicData={studentStatus ?? ''} description="Status ucznia/studenta: " />
```

Comment:
a co w przypadku gdy którejś z tych informacji nie będzie?

Reply:
oooo zapomniałam, thx !

Reply:
jak to? któraś jest nieobowiązkowa?

Reply:
jak się dopiero co user zarejestruje i nie wypełnił wszystkiego to nadal może podejrzeć CV bez pewnych elementów